README.md

```diff
@@ -44,17 +44,61 @@ normalization logic.
 
 ### Run & Stop
 
-        $ ./tempesta.sh start
-        $ ./tempesta.sh stop
+Use ```tempesta.sh``` script to run and stop Tempesta. The script provides help
+info by ```--help``` switch. Usage example:
+
+        $ ./tempesta.sh --start
+        $ ./tempesta.sh --stop
+
 
 ### Configuration
 
 Tempesta is configured via plain-text configuration file.
 
-The file location is determined by the `TFW_CFG_PATH` environment variable:
+The file location is determined by the `TFW\_CFG\_PATH` environment variable:
 
         $ TFW_CFG_PATH="/opt/tempesta.conf" ./tempesta.sh start
 
-By default, the `tempesta_fw.conf` from this directory is used.
+By default, the `tempesta\_fw.conf` from this directory is used.
+
+See ```tempesta\_fw.conf` for the list of available options and their descriptions.
```

Comment:
Triple opening backquote and a single closing backquote. Just use single backquotes.
Also, "description", not "descriptions".